docassemble/mlhframework/data/questions/mlh_interview_framework.yml

```diff
@@ -192,28 +192,37 @@ continue button field: MLH_intro_saving_answers
 id: MLH intro PII
 question: Private Information
 subquestion: |
-  To protect privacy, you **cannot** include personal identifying information in public documents that are filed in court. Personal identifying information includes:
+  To protect privacy, you **cannot** include personal identifying information in public court documents. Personal identifying information includes:
   
   * Date of birth
   * Social security or national identification number
   * Driver’s license or state identification number
   * Passport number
   * Financial account numbers
 
-  **If a question doesn't specifically ask for personal identifying information, do not include it in your answers.** 
-  When this information is necessary, this tool will specifically ask for the information and it will go on a separate non-public form that you will file along with the other documents. 
-  This DIY tool will automatically produce that form if you need it.
+  **Don’t** put any personal identifying information in your answers **unless we specifically ask for it.** 
```

Comment:
Good streamlining. I have two suggestions;
1. Use "**Do not**" instead of "**Don't**" (seems a little stronger)
2. Change end to "**unless you are specifically asked for it**" (I have always found it a little strange to use "we" in these interviews, even when I myself have done it that way)

Reply:
Sounds good to me on 1. On 2, what about "**unless this tool specifically asks for it.**"  We've been told to avoid passive verbs when possible, and that way it'll match the final question.

Reply:
Sounds good to me!